diff -ur tmp/bearssl-0.6/conf/Unix32.mk bearssl-0.6/conf/Unix32.mk
--- tmp/bearssl-0.6/conf/Unix32.mk	2018-08-14 22:41:54.000000000 +0200
+++ bearssl-0.6/conf/Unix32.mk	2023-08-11 10:48:33.662098719 +0200
@@ -7,6 +7,6 @@
 include conf/Unix.mk
 
 BUILD = build32
-CFLAGS = -W -Wall -Os -fPIC -m32 -DBR_LOMUL
+CFLAGS = -W -Wall -g -Os -fPIC -m32 -DBR_LOMUL
 LDFLAGS = -m32
 LDDLLFLAGS = -shared -m32
diff -ur tmp/bearssl-0.6/conf/Unix.mk bearssl-0.6/conf/Unix.mk
--- tmp/bearssl-0.6/conf/Unix.mk	2018-08-14 22:41:54.000000000 +0200
+++ bearssl-0.6/conf/Unix.mk	2023-08-11 10:48:42.389828384 +0200
@@ -38,7 +38,7 @@
 
 # C compiler and flags.
 CC = cc
-CFLAGS = -W -Wall -Os -fPIC
+CFLAGS = -W -Wall -g -Os -fPIC
 CCOUT = -c -o 
 
 # Static library building tool.
